bin.js

```diff
@@ -31,3 +34,12 @@ function getConfig() {
   }
 }
 
+function getRules(set) {
+  var configFileName = set ? ('eslint-config-' + set) : null
```

Comment:
> I just wanted to be sure, we make the same assumptions here.

We should probably document this limitation though, because other people make other assumptions :-)

Yeah, I think you've described the desired behavior. We'll probably run into some issues with those side-effects. But let's see how far we can get :-)